config/routes.rb

```diff
@@ -214,6 +214,7 @@
   get 'u/:username/'                       => 'users#show',       as: :user
 
   get 'dashboard'                          => 'application#boot_angular_ui', as: :dashboard
+  get 'dashboard/show_muted'               => 'application#boot_angular_ui', as: :muted
```

Comment:
I think this wants to be `get dashboard/:filter`, which will allow navigation to any dashboard filter we come up with, not just muted.